src/main/kotlin/org/rust/cargo/project/configurable/RsExternalLinterConfigurable.kt

```diff
@@ -11,33 +11,43 @@ import com.intellij.ui.EnumComboBoxModel
 import com.intellij.ui.dsl.builder.*
 import org.rust.RsBundle
 import org.rust.cargo.project.model.cargoProjects
+import org.rust.cargo.project.settings.RsExternalLinterProjectSettingsService
+import org.rust.cargo.project.settings.externalLinterSettings
 import org.rust.cargo.toolchain.ExternalLinter
 import org.rust.cargo.util.CargoCommandCompletionProvider
 import org.rust.cargo.util.RsCommandLineEditor
 import org.rust.openapiext.fullWidthCell
 
 class RsExternalLinterConfigurable(project: Project) : RsConfigurableBase(project, RsBundle.message("settings.rust.external.linters.name")) {
+    private val settings: RsExternalLinterProjectSettingsService = project.externalLinterSettings
+
+    private val additionalArguments: RsCommandLineEditor =
+        RsCommandLineEditor(project, CargoCommandCompletionProvider(project.cargoProjects, "check ") { null })
+
     override fun createPanel(): DialogPanel = panel {
-        row(RsBundle.message("settings.rust.external.linters.tool.label")) {
-            comboBox(EnumComboBoxModel(ExternalLinter::class.java))
-                .comment(RsBundle.message("settings.rust.external.linters.tool.comment"))
-                .bindItem(state::externalLinter.toNullableProperty())
-        }
+        group(indent = false) {
```

Comment:
It seems it adds unexpected group separator

<img width="1094" alt="Screenshot 2023-02-22 at 12 41 06" src="https://user-images.githubusercontent.com/2539310/220610112-1dbef959-3210-46d8-bd26-bd1a82925095.png">